src/Sentry/Sentry.csproj

```
@@ -29,7 +29,9 @@
       <Link>%(RecursiveDir)\%(Filename)%(Extension)</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup Condition="$(TargetFramework.StartsWith('netstandard')) or $(TargetFramework.StartsWith('net4'))">
+
+  <!-- Ben.Demystifier also needs System.Reflection.Metadata 5.0.0 or higher on all platforms. -->
+  <ItemGroup Condition="$(TargetFramework.StartsWith('netstandard')) or $(TargetFramework.StartsWith('net4')) or $(TargetFramework.StartsWith('netcoreapp'))">
     <PackageReference Include="System.Reflection.Metadata" Version="5.0.0" />
```

Just a suggestion 🙂
One of the challenges with standalone comments is that those are not linked/associated with the package reference. An accidental re-arrangement in the CSPROJ can sever the connection. `PackageReference` has a `Label` property that could be used to keep the connection between the comment and the package.

```suggestion
  <ItemGroup Condition="$(TargetFramework.StartsWith('netstandard')) or $(TargetFramework.StartsWith('net4')) or $(TargetFramework.StartsWith('netcoreapp'))">
    <PackageReference Include="System.Reflection.Metadata" Version="5.0.0" Label="Ben.Demystifier also needs System.Reflection.Metadata 5.0.0 or higher on all platforms." />
```

---

Thanks, but I believe `Label` isn't appropriate for lengthy comments.  It becomes metadata on the item that can be used by other `Condition` expressions.  The MS docs are poor on this also - they've documented `ItemGroup.Label`, but not `Item.Label`.  Either way, examples I've seen would use this for shorter strings like `Label="Something"`, rather than comments.